pyciemss/interfaces.py

```diff
@@ -627,13 +627,13 @@ def calibrate(
             - The start time of the model. This is used to align the `start_state` from the
               AMR model with the simulation timepoints.
             - By default we set the `start_time` to be 0.
-        static_state_interventions: Dict[float, Dict[str, Intervention]]
+        static_state_interventions: Dict[torch.Tensor, Dict[str, Intervention]]
             - A dictionary of static interventions to apply to the model.
             - Each key is the time at which the intervention is applied.
             - Each value is a dictionary of the form {state_variable_name: intervention_assignment}.
             - Note that the `intervention_assignment` can be any type supported by
               :func:`~chirho.interventional.ops.intervene`, including functions.
-        static_parameter_interventions: Dict[float, Dict[str, Intervention]]
+        static_parameter_interventions: Dict[torch.Tensor, Dict[str, Intervention]]
```

Comment:
Is this change necessary for this PR or is it separate?

Reply:
It is a correction. There is a mismatch between what is in the function definition and the documentation. For this small of a change, I think it is better to fix it now.